packages/js/src/components/contentAnalysis/InclusiveLanguageAnalysis.js

```diff
@@ -84,7 +84,7 @@ const InclusiveLanguageAnalysis = ( props ) => {
 				<Results
 					results={ props.results }
 					marksButtonClassName="yoast-tooltip yoast-tooltip-w"
-					marksButtonStatus={ props.marksButtonStatus }
+					marksButtonStatus={ props.shortcodesForParsing.length > 0 ? "disabled" : props.marksButtonStatus }
```

Comment:
Is it possible to do this check in `getMarkButtonStatus()`, so you don't have to repeat it across the `*Analysis.js` files?

Reply:
I am not sure if it's a good idea to add an extra logic inside `getMarkButtonStatus` selector. I thought that generally we want to make sure that selectors, actions, and reducers are pure function, which means that we should return what is in the state? At least that's what I learned from articles like [this one](https://dev.mobify.com/v2.x/how-to-guides/categories/state-management/getting-data-out-of-the-redux-store-with-selectors) 🤔

Reply:
I moved this logic to `packages/js/src/components/contentAnalysis/Results.js`. This way we only need to do this check once 😺